<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.61, y2:0.086, stop:0 rgba(46, 50, 83, 255), stop:0.751412 rgba(128, 50, 148, 255));border:0px solid red;
border-radius:6px;}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>112</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(255, 25, 25);</string>
        </property>
        <property name="text">
         <string>Author:Jecjiang</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(20, 255, 8);</string>
        </property>
        <property name="text">
         <string>811365949@qq.com</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(255, 25, 25);</string>
        </property>
        <property name="text">
         <string>翻页滑屏</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(20, 255, 8);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(20, 255, 8);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/new/prefix1/Resourse/appbar.chevron.left.png);}

QPushButton:hover{border-image: url(:/new/prefix1/Resourse/appbar.chevron.left.png);background-color: rgb(137, 255, 46,90);}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AnimationStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 255, 17,0);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 85, 187);
border-radius:6px;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout"/>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(83, 91, 255);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 255, 17,0);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(83, 91, 255);
border-radius:6px;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2"/>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 16, 20);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(12, 36, 255);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 147, 133);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 255, 17,0);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(83, 91, 255);
border-radius:6px;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3"/>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(157, 255, 130);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="widget_9" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(11, 255, 255);
border-radius:6px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/new/prefix1/Resourse/appbar.chevron.right.png);}

QPushButton:hover{border-image: url(:/new/prefix1/Resourse/appbar.chevron.right.png);background-color: rgb(137, 255, 46,90);}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(24, 46, 58, 90);
font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(255, 25, 25);</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255,90);
color: rgb(255, 25, 25);
font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255,90);
color: rgb(255, 25, 25);
font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
